modules/admin-ui-picard/app/src/components/shared/wizard/WizardStepper.js

```diff
@@ -10,8 +10,9 @@ import { connect } from 'react-redux';
 /**
  * This components renders the stepper navigation of new resource wizards
  */
-const WizardStepper = ({ steps, page, setPage, formik, hasAccessPage=false, checkAcls,
-    completed, setCompleted }) => {
+const WizardStepper = ({ steps, page, setPage, formik, checkAcls, completed, setCompleted,
```

Comment:
Please move checkAcls to last place in list of props, because it is the only one that comes from mapDispatchToProps, the others are set in the parent components (or use the default value)